server/utils/isRequired.util.js

```diff
@@ -0,0 +1,20 @@
+
+/**
+ * @desciption - Specify a key of an array that is required in another array
+ *
+ * @param {string} haystack
+ * @param {string} needle
+ * @returns {string} slug
+ */
+const isRequired = (haystack, needle) => {
+  const required = [];
+  // eslint-disable-next-line array-callback-return
```

Comment:
Check out this resource https://eslint.org/docs/rules/array-callback-return and revert accordingly

Reply:
Ok, will implement now.

Reply:
This has been implemented.